package.json

```diff
@@ -5,8 +5,9 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "prod": "pnpm run migrate && tsx --env-file-if-exists=.env --dns-result-order=ipv6first src/index.tsx",
-    "dev": "pnpm run migrate && tsx watch --env-file-if-exists=.env --dns-result-order=ipv6first src/index.tsx",
+    "prod": "pnpm run migrate && tsx --env-file-if-exists=.env --dns-result-order=ipv6first bin/server.ts",
+    "dev": "pnpm run migrate && tsx watch --env-file-if-exists=.env --dns-result-order=ipv6first bin/server.ts",
+    "test": "tsx --env-file-if-exists=.env.test --test",
     "check": "tsc && biome check .",
     "migrate": "drizzle-kit migrate",
```

Comment:
We probably need to use `dotenv` in drizzle config in order to support migrating the test database in development, in CI the need for dotenv is bypassed because we're setting the `DATABASE_URL` as an environment variable.